src/db/migrations/14-add-hume-angelbaby-otherside.ts

```diff
@@ -0,0 +1,40 @@
+
+import { Knex } from 'knex';
+
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { addPlatform, addNftFactory, removeNftFactory, removePlatform } from '../migration-helpers';
+
+const HUME_PLATFORM: MusicPlatform = {
+  id: 'hume',
+  type: MusicPlatformType['single-track-multiprint-contract'],
```

Comment:
hmm this type expects all nfts on the platform to be the same artist. currently hume only has 1 artist but i imagine they could add more in future. not sure if this will work long term, still looking at it..

Reply:
i guess we can just override the artist id too?

Reply:
yeah, I guess we could also use `getTrait` to read the artist value from the metadata but I'm not sure how generic/well supported that would be?

Reply:
yeh, it's fairly common. it's maybe a better default to use than defaulting to the platformId which is what we currently do, but still will need manual checks nonetheless